FirebaseAuthUI/AccountManagement/FUIAccountSettingsOperationForgotPassword.m

```diff
@@ -65,30 +65,26 @@ - (void)onPasswordRecovery:(NSString *)email {
 
   [self.delegate.auth sendPasswordResetWithEmail:email
                              completion:^(NSError *_Nullable error) {
-    // The dispatch is a workaround for a bug in FirebaseAuth 3.0.2, which doesn't call the
-    // completion block on the main queue.
-    dispatch_async(dispatch_get_main_queue(), ^{
-      [self.delegate decrementActivity];
```

Comment:
If we're removing this, we should bump the Podfile dependency to 3.1 or higher so this bug doesn't regress for users who haven't updated their pods.

Reply:
we already have 4.2 as minimum requirement https://github.com/firebase/FirebaseUI-iOS/blob/master/FirebaseUI.podspec#L50